ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/service-routing.yml

```diff
@@ -0,0 +1,5 @@
+#### service-routing ####
+
+service_routing_configuration:
+  bgp_no_equals_default: true
+service_routing_protocols_model: multi-agent
```

Comment:
remove `service_routing_protocols_model: multi-agent` as it should stay in it's own dedicated test to avoid confusion as it's two separate templates.